test/tests/multiapps/sub_cycling/master_sub_output.i

```diff
@@ -0,0 +1,68 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+[]
+
+[Variables]
+  [./u]
+  [../]
+[]
+
+[Kernels]
+  [./diff]
+    type = Diffusion
+    variable = u
+  [../]
+  [./td]
+    type = TimeDerivative
+    variable = u
+  [../]
+[]
+
+[BCs]
+  [./left]
+    type = DirichletBC
+    variable = u
+    boundary = left
+    value = 0
+  [../]
+  [./right]
+    type = DirichletBC
+    variable = u
+    boundary = right
+    value = 1
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  num_steps = 4
+  dt = 0.1
+
+  # Preconditioned JFNK (default)
+  solve_type = 'PJFNK'
+
+  petsc_options_iname = '-pc_type -pc_hypre_type'
+  petsc_options_value = 'hypre boomeramg'
+[]
+
+[Outputs]
+  output_initial = true
+  exodus = true
+  print_linear_residuals = true
+  print_perf_log = true
+[]
+
+[MultiApps]
+  [./sub]
+    type = TransientMultiApp
+    app_type = MooseTestApp
+    execute_on = timestep_end
+    positions = '0 0 0'
+    input_files = sub.i
+    sub_cycling = true
+    output_sub_cycles = true
```

Comment:
To be more clear: I created that option to get file output from intermediate steps.  That's what it did.

If it is now necessary to use that option to get Console output: that is a bug.

The reason is the normal reason: we normally want Console output - but almost always _don't_ want the intermediate file output... so that should be the default.